msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -213,6 +221,15 @@ private void checkCurrentAccountNotifyCallback(@NonNull final CurrentAccountCall
     }
 
 
+    /**
+     * @param activity Non-null {@link Activity} that is used as the parent activity for launching the {@link AuthenticationActivity}.
+     * @param scopes   The non-null array of scopes to be consented to during sign in.
+     *                 MSAL always sends the scopes 'openid profile offline_access'.  Do not include any of these scopes in the scope parameter.
+     *                 The access token returned is for MS Graph and will allow you to query for additional information about the signed in account.
+     * @param callback {@link AuthenticationCallback} that is used to send the result back. The success result will be
+     *                 sent back via {@link AuthenticationCallback#onSuccess(IAuthenticationResult)}.
+     *                 Failure case will be sent back via {
+     */
```

Comment:
failure case will be sent back via what?

Reply:
Fixed. Thanks